src/assets/templates/academics_semester_classes.html

```diff
@@ -195,21 +195,35 @@ <h3>Swaps</h3>
             <tbody>
             <tr data-ng-repeat="swapClass in swapClassList">
               <td>
-                <span class="cc-academics-semester-class-swap-number cc-academics-semester-class-swap-number-width">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-academics-semester-class-swap-number-small-padding">&#x21C4;</span>
-              </td>
+                <span class="cc-academics-semester-class-swap-number cc-academics-semester-class-swap-number-width">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-academics-semester-class-swap-number-small-padding" aria-hidden="true">&#x21C4;</span>
+                <span class="cc-visuallyhidden">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-visuallyhidden"> Swap</span> </td>
```

Comment:
@frankiepun ,
Could this be reduced to?
`<span class="cc-visuallyhidden">{{swapClass.swapFromCourse.swap_number}} Swap</span>`

Reply:
Excellent feedback. Thanks for pointing it out. I combined the span tags.